crates/testing-framework/src/execution/mod.rs

```diff
@@ -15,23 +15,41 @@
  */
 
 use crate::{
-    asserts::ServiceDefinition,
-    ephemeral::{Network, Peer, PeerId},
-    services::{results::ResultService, MarineService, MarineServiceHandle},
-    transform::{walker::Transformer, Sexp},
+    ephemeral::{Network, PeerId},
+    queue::ExecutionQueue,
+    services::MarineServiceHandle,
+    transform::walker::Transformee,
 };
 
 use air_test_utils::{test_runner::TestRunParameters, RawAVMOutcome};
 
-use std::{borrow::Borrow, collections::HashMap, hash::Hash, rc::Rc, str::FromStr};
+use std::{borrow::Borrow, hash::Hash, rc::Rc};
 
 pub struct TestExecutor {
-    pub air_script: String,
-    pub network: Network,
-    pub test_parameters: TestRunParameters,
+    transformee: Transformee,
+    test_parameters: TestRunParameters,
```

Comment:
`TestRunParameters` is not a property of a test network, but a property of particular script launch, so it should be passed only on script running

Reply:
You may always create many test executors with different test_parameters, the behavior will be exactly the same, though not as explicit.

This particular design prevents using data from one particle with different particle, as the `queue` is bundled with the `test_parameters`.